tests/messages-request-log-subagent.test.ts

```
@@ -99,6 +103,49 @@ function buildAnthropicResponse(model: string, text: string) {
+  fetchHolder.fetch = fetchMock
+}
+
+async function withConfig(
+  config: {
+    modelAliases: Record<string, string>
+    allowOriginalModelNamesForAliases: boolean
+    smallModel: string
+  },
+  run: () => Promise<void>,
+): Promise<void> {
+  const original = await fs.readFile(PATHS.CONFIG_PATH, "utf8").catch(() => null)
```

ESLint

[error] Replace `.readFile(PATHS.CONFIG_PATH,·"utf8")` with `⏎····.readFile(PATHS.CONFIG_PATH,·"utf8")⏎····`

(prettier/prettier)